core/src/main/resources/lib/form/password.jelly

```diff
@@ -61,12 +61,73 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
   <f:prepareDatabinding />
-  <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
-         class="setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
-         name ="${attrs.name ?: '_.'+attrs.field}"
-         value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"
-         type="password"
-         checkMethod="post"
-         ATTRIBUTES="${attrs}" EXCEPT="field clazz value" />
-  <!-- TODO consider customizedFields -->
+  <j:choose>
+    <j:when test="${h.useHidingPasswordFields()}">
+      <j:set var="value" value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"/>
```

Comment:
I think the name `value` is misleading here. Better use `passwordValue` or similar. Otherwise we could suppose it's just the plain text value, which is wrong as it's the encrypted value or `********`.